demonhunter/nodes/manager/__init__.py

```diff
@@ -1,25 +1,43 @@
 from aiohttp import web
+import ssl
+import os
 
 from demonhunter.nodes.manager.app import add_routes, run_app
 from demonhunter.nodes.manager.agent.server import AgentManager
 from demonhunter.core.loggers.logfile import FileLogger
 
 
 class Manager:
-
     agents = list()
-    def __init__(self, loop, server_address, webapp_address, agent_manager=True,
-                 web_app=None, agent_password=None, logfile=None,
-                 port=16742):
+
+    def __init__(self, loop, server_address, webapp_address, port, certificate, key, agent_manager=True,
+                 web_app=None, agent_password=None, logfile=None):
         self.server_address = server_address
         self.webapp_address = webapp_address
         self.loop = loop
         self.port = port
+        self.certificate = certificate
+        self.key = key
+
+        # configure communication via ssl if the certificate and key are present
+        if self.certificate is not None and self.certificate is not None:
```

Comment:
why did you use an expression twice, is it a special python syntax? or a logic that i cant undrestand ?